jekyll/_style/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: classic-docs
+title: "Welcome to CircleCI Style Documentation"
+description: "Welcome to CircleCI Style Documentation"
+permalink: style/
+toc: false
+---
+
+This guide is a reference for contributors to CircleCI docs. Our style guide, like language itself, is constantly evolving so you will probably find older documentation does not meet the requirements set out here. Use this guide to help you write new docs, and when working on existing documentation it would be great if you can make edits to help us get older documents to conform to.
```

Comment:
```suggestion
This guide is a reference for contributors to CircleCI docs. Our style guide evolves so you will probably find older documentation does not meet the requirements set out here. Use this guide to help you write new docs, and when working on existing documentation it would be great if you can make edits to help us get older documents to conform to.
```